Header header
int8 basic_ctrl
int8 target_node
float32 target_heading
int8 route_segs_num
int8[50] segs_vector
